### ISSUE TITLE
Add reload button in pull tab

### ISSUE THREAD

**Issue by ggalmazor** [opendatakit/briefcase#612]
We should add a button on the pull tab to let users reload the forms list in case new forms are available.

This is a mockup of such a button:
![image](https://user-images.githubusercontent.com/205913/44463979-bb34e800-a619-11e8-9d55-bda852eb74e1.png)

**Comment by jpknox** [comment]
I was having a poke at this but I became blocked by an error - I can't pull data from an aggregate server. The URL in your picture gives me an SSL error and the URL from the tutorial page (https://opendatakit.appspot.com/) gives me the following error readout.

`Fetching form definition
resolving against briefcase form definitions
Error parsing form definition: org.opendatakit.aggregate.exception.ODKIncompleteSubmissionData: Javarosa failed to construct a FormDef. Is this an XForm definition?`

**Comment by jpknox** [comment]
Glad to be back @yanokwa!

The SSL error has disappeared for myself.

Your suggestion is working well for me so far. May I continue using it for this issue?

@opendatakit-bot claim

**Comment by jpknox** [comment]
@ggalmazor I recall that you suggested a minor tweak to the functionality of the button. Would you prefer to include it in this ticket or in a new one instead?

**Comment by ggalmazor** [comment]
Hi, @jpknox!

I believe we're talking about the lack of user feedback when the reload process finishes (whether it's successful or there's been some error).

I think a new issue would be best for this.

**Comment by ggalmazor** [comment]
Hi, @jpknox!

I'm working on #595, which is somewhat affected by the new reload button you worked on. I think I could take on the missing user feedback we were talking about in the previous comments, which would solve #595.

Have you started working on this already? Are you planning to? If not, I'd like to take it right away, if that's OK.

**Comment by ggalmazor** [comment]
Hi, @jpknox!

I went ahead and created a PR #643.

I'm going to close this issue, by the way. It should have been closed after merging #620, I think.

**Comment by Rutayuga** [comment]
Dear @jpknox and @yanokwa  How did you solve this Error below ? I cannot pull data from Briefcase  
Fetching form definition
resolving against briefcase form definitions
Error parsing form definition: org.opendatakit.aggregate.exception.ODKIncompleteSubmissionData: Javarosa failed to construct a FormDef. Is this an XForm definition?

**Comment by ggalmazor** [comment]
Hi, @Rutayuga!

We're already trying to figure your issue out [in the forums](https://forum.opendatakit.org/t/fetching-form-definition-resolving-against-briefcase-form-definitions-error-parsing-form-definition-org-opendatakit-aggregate-exception-odkincompletesubmissiondata-javarosa-failed-to-construct-a-formdef-is-this-an-xform-definition/20160). Please, let's move the conversation there.
